pyani_plus/methods/method_anib.py

```diff
@@ -40,19 +40,18 @@
 MIN_COVERAGE = 0.7
 MIN_IDENTITY = 0.3
 
-# We do NOT use the standard 12 columns, but a custom 15 cols as per old pyANI
 # std = qaccver saccver pident length mismatch gapopen qstart qend sstart send evalue bitscore
-BLAST_COLUMNS = (
-    "qseqid sseqid length mismatch pident nident"
-    " qlen slen qstart qend sstart send positive ppos gaps"
-).split()
+# We do NOT use the standard 12 columns, nor a custom 15 cols as per old pyANI,
+# but the minimal set of only the 6 used fields from those 15.
+BLAST_COLUMNS = ("qseqid pident length mismatch qlen gaps").split()
 
 # Precompute the column indexes once
+BLAST_COL_QUERY = BLAST_COLUMNS.index("qseqid")  # alternative to qaccver in std
```

Comment:
It seems using either ``qseqid`` or ``qaccver`` will give the expected values like ``frag00001`` but hypothetically one could be fractionally faster than the other in BLAST's TSV output?